homeassistant/components/shell_command/__init__.py

```diff
@@ -97,16 +103,24 @@ async def async_service_handler(service: ServiceCall) -> None:
                     process._transport.close()  # type: ignore[attr-defined]
                 del process
 
-            return
+            return None
```

Comment:
Can we add a test covering this?

Reply:
Added check to existing test in last commit. But as this test is disabled (`disabled to check if it fixes flaky CI`), its not counted by the test coverage.

Reply:
Oh, I see.
The test is fixed in <https://github.com/home-assistant/core/pull/96834>, please rebase this PR once that has been merged 👍